Documentation/Usage.md

```diff
@@ -0,0 +1,134 @@
+
+# Usage
+
+<br>
+
+## Configuration
+
+Add the following to your `package.json` :
+
+<br>
```

Comment:
I prefer to use simple markdown instead of HTML in markdown.

Reply:
Markdown is an extension to HMTL not the other way around.

Reply:
`<br>` is not required once we have `enter` in Markdown.